### PR TITLE
Enable DFE Analytics entity table check job

### PR DESCRIPTION
**Pull request by spencerldixon**
### Trello card

https://trello.com/c/zG6KY0HX/5540-update-dfe-analytics-on-git-and-switch-on-an-entity-table-check

### Context

The Data Insights team would like us to enable an [Entity Table Check job](https://github.com/DFE-Digital/dfe-analytics/blob/main/lib/dfe/analytics/entity_table_check_job.rb) on GIT which sends a row_count, a checksum and a timestamp for each entity table sent to BigQuery

This helps them to confirm that database tables match BigQuery and helps them debug them where they don’t

### Changes proposed in this pull request

- Update the DFE Analytics gem
- Install and configure the `sidekiq-cron` gem for running recurring sidekiq jobs
- Configure the DFE Analytics entity table check job to run every evening at 00:30

### CODE REVIEW

**Comment by martyn-w** [comment]
👍  LGTM - however Spencer has identified a version incompatibility between Redis and Sidekiq on production, so lets wait for that to be resolved before merging this. Also would be good to get a review from the Data Insights / Analytics team.

**Review by martyn-w — APPROVED** [review]

**Review by ericaporter — APPROVED** [review]
Looks good 👍

**Review by martyn-w — APPROVED** [review]

**Pull request merged**
